prov/efa/src/efa_mr.c

```diff
@@ -57,6 +57,58 @@ static struct fi_ops efa_mr_cache_ops = {
 	.ops_open = fi_no_ops_open,
 };
 
+/*
+ * @brief Validate HMEM attributes and populate efa_mr struct
+ *
+ * Check if FI_HMEM is enabled for the domain, validate whether the specific
+ * device type requested is currently supported by the provider, and update the
+ * efa_mr structure based on the attributes requested by the user.
+ *
+ * @params[in]	efa_mr	efa_mr structure to be updated
+ * @params[in]	attr	fi_mr_attr from the user's registration call
+ *
+ * @return FI_SUCCESS or negative FI error code
+ */
+static int efa_ep_mr_hmem_setup(struct efa_mr *efa_mr,
```

Comment:
nit: this function is not related to ep, so the name can be just `efa_mr_hmem_setup`.

Reply:
Good point, fixed.